setup.py

```diff
@@ -207,13 +209,13 @@ def run(self):
 
         add_ext_cflags(ext, compiler)
 
-        if compiler.compiler_type == "msvc":
-            ext.libraries += ['cairo']
-        else:
-            pkg_config_version_check('cairo', CAIRO_VERSION_REQUIRED)
-            ext.include_dirs += pkg_config_parse('--cflags-only-I', 'cairo')
-            ext.library_dirs += pkg_config_parse('--libs-only-L', 'cairo')
-            ext.libraries += pkg_config_parse('--libs-only-l', 'cairo')
+        pkg_config_version_check('cairo', CAIRO_VERSION_REQUIRED)
```

Comment:
just wondering, why do we need pkg-config now with msvc?

Reply:
> why do we need pkg-config now with msvc

Since now we're linking to a Cairo as a static library finding the dependency would be difficult to maintain manually. Also, users (like say people from conda) can build without having to set any extra environmental variables.

I will probably make this check optional so it uses pkg-config when it's there. Also, are there people building pycairo with msvc other than projects like conda and gvs-build?

Reply:
OK, yeah, hard to say... I remember https://github.com/pygobject/pycairo/pull/72 for example, but that was before cairo was using meson, so .pc files are now a thing.

Maybe some env vars: `PYCAIRO_BUILD_NO_PKGCONF`, `PYCAIRO_BUILD_STATIC` ?

gvsbuild could probably move to meson if that doesn't work for them.

Reply:
> Maybe some env vars: PYCAIRO_BUILD_NO_PKGCONF, PYCAIRO_BUILD_STATIC ?

Looks good, will look at it later today.

Reply:
Done https://github.com/pygobject/pycairo/pull/243/commits/d3fe107645b27ab0c8d8e286f1abdc3f3eb309c0